airbyte-integrations/connectors/source-zendesk-support/unit_tests/test_other.py

```diff
@@ -6,20 +6,59 @@
 from datetime import datetime
```

Comment:
is there a better name for this test rather than `test_other.py`? ;)

Reply:
"unit_tests" ? :) Didn't find anything better than "other", because it's a collection of different tests for different methods.